core/src/com/unciv/logic/map/UnitMovementAlgorithms.kt

```diff
@@ -343,7 +343,14 @@ class UnitMovementAlgorithms(val unit:MapUnit) {
         }
         return false
     }
-
+    
+    // Can a paratrooper land at this tile?
+    fun canParadropOn(destination: TileInfo): Boolean {
+        // Can only move to land tiles within range that are visible and not impassible
+        // Based on some testing done in the base game
+        if (!destination.isLand || destination.isImpassible() || !unit.civInfo.viewableTiles.contains(destination)) return false
```

Comment:
`return destination.isLand && !destination.isImpassible() && unit.civInfo.viewableTiles.contains(destination)` seems more readable to me :eyes:  - shouldn't the automatic inspections whine about such things?